executor/show.go

```diff
@@ -996,6 +1006,58 @@ func (e *ShowExec) fetchShowWarnings(errOnly bool) error {
 	return nil
 }
 
+// fetchShowPumpStatus gets status of all pumps and fill them into e.rows.
+func (e *ShowExec) fetchShowPumpStatus() error {
```

Comment:
we can combine function `fetchShowPumpStatus` and `fetchShowDrainerStatus`

Reply:
Done.